backend/clubs/management/commands/stress_test.py

```diff
@@ -0,0 +1,173 @@
+        clubs = Club.objects.filter(code__startswith=self.club_prefix)
+
+        # Create Club Applications
+        applications = [
+            ClubApplication(
+                name="Test Application",
+                club=club,
+                application_start_time=now - datetime.timedelta(days=1),
+                application_end_time=now + datetime.timedelta(days=3),
+                result_release_time=now + datetime.timedelta(weeks=1),
+                external_url="https://pennlabs.org/",
+            )
+            for club in clubs
+        ]
+        ClubApplication.objects.bulk_create(applications)
+        applications = ClubApplication.objects.filter(club__in=clubs)
+
+        # Create Club Application Questions
+        questions = [
+            ApplicationQuestion(
+                question_type=ApplicationQuestion.FREE_RESPONSE,
+                prompt="Answer the prompt you selected",
+                word_limit=150,
+                application=application,
+            )
+            for _ in range(self.num_questions_per_club)
+            for application in applications
+        ]
+        ApplicationQuestion.objects.bulk_create(questions)
+
+        clubs_data = Club.objects.filter(
+            code__startswith=self.club_prefix
+        ).prefetch_related(
+            Prefetch(
+                "clubapplication_set",
+                queryset=ClubApplication.objects.prefetch_related("questions"),
+            )
+        )
+        for club in clubs_data:
+            question_ids = [
+                str(question.id)
+                for application in club.clubapplication_set.all()
+                for question in application.questions.all()
+            ]
+            self.club_question_ids[club.id] = question_ids
+        print("Finished setting up clubs.")
+
+        # Create Users (Applicants)
+        User = get_user_model()
+        User.objects.bulk_create(
+            [
+                User(
+                    username=self.user_prefix + str(i),
+                    email=str(i) + "@upenn.edu",
+                    password="test",
+                )
+                for i in range(self.num_users)
+            ]
+        )
+        self.users = list(User.objects.filter(username__startswith=self.user_prefix))
+        print("Finished setting up users.")
+
+    @sync_to_async
+    def submit_application(self, user, club_id):
+        start_time = time.time()
+        data = {
+            "questionIds": self.club_question_ids[club_id],
+        }
+        for question_id in self.club_question_ids[club_id]:
+            data[question_id] = {"text": "This is a test answer."}
+
+        request = self.factory.post(self.uri, data, format="json")
+        request.user = user
+
+        self.view(request)
+        end_time = time.time()
+        return end_time - start_time
+
+    def tearDown(self):
+        Club.objects.filter(code__startswith=self.club_prefix).delete()
```

Comment:
Might have stray `ClubApplication`, `ApplicationSubmission`, etc objects which you'll need to tear down

Reply:
Checking from the models, this should all be handled with `on_delete` behavior? It should form separate "trees" with roots of Clubs and Users.

Reply:
Sure, this makes sense. Please do double check empirically though; we should definitely avoid excess footprint on the DB, especially if we test in prod multiple times.

Reply:
Yeah just double checked, no rows are left over. The only concern I guess could be the autoincrementing index and offsetting that by the number of test instances. Should not be an issue at all, but if we want full isolation, we could create a separate database on the same instance.

Reply:
I think this should be fine! Thank you for double checking